package.json

```diff
@@ -19,7 +19,7 @@
     "xml-crypto": "^3.0.0"
   },
   "peerDependencies": {
-    "axios": "^0.27.2"
+    "axios": ">=1.5.0"
```

Comment:
Still not sure why this is a peer dependency. This module CAN NOT function without axios, it clearly is a dependency, lol

Reply:
https://github.com/vpulim/node-soap/pull/1179

Reply:
That point is no longer valid. this package can easily state that it directly depends on axios >= 1.5.0

In my project I would be able to use any axios version that satisfies the constraint